application/comit_node/src/http_api/rfc003/swap.rs

```diff
@@ -284,19 +283,18 @@ fn handle_get_swap<T: MetadataStore<SwapId>, S: StateStore<SwapId>>(
     let metadata = metadata_store
         .get(id)?
         .ok_or_else(problem::swap_not_found)?;
-    get_swap!(
+
+    with_swap_types!(
         &metadata,
-        state_store,
-        id,
-        state,
         (|| {
-            let state =
-                state.ok_or_else(|| HttpApiProblem::with_title_and_type_from_status(500))?;
+            trace!("Fetched metadata of swap with id {}: {:?}", id, metadata);
```

Comment:
🔧  If we want to trace log this let's put it in `MetadataStore::get` and remove this one.